cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -35,21 +34,28 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	clusterctlv1 "sigs.k8s.io/cluster-api/cmd/clusterctl/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
-	manifests "sigs.k8s.io/cluster-api/cmd/clusterctl/config"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/scheme"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/test"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// Those values are dummy for test only
+	expectedHash    = "dummy-hash"
+	expectedVersion = "v0.11.2"
+)
+
 func Test_VersionMarkerUpToDate(t *testing.T) {
-	yaml, err := manifests.Asset(embeddedCertManagerManifestPath)
-	if err != nil {
-		t.Fatalf("Failed to get cert-manager.yaml asset data: %v", err)
+	pollImmediateWaiter := func(interval, timeout time.Duration, condition wait.ConditionFunc) error {
+		return nil
 	}
+	fakeConfigClient := newFakeConfig("")
+	cm, err := newCertManagerClient(fakeConfigClient, nil, pollImmediateWaiter)
 
-	actualHash := fmt.Sprintf("%x", sha256.Sum256(yaml))
 	g := NewWithT(t)
-	g.Expect(actualHash).To(Equal(embeddedCertManagerManifestHash), "The cert-manager.yaml asset data has changed, but embeddedCertManagerManifestVersion and embeddedCertManagerManifestHash has not been updated.")
+	g.Expect(err).ToNot(HaveOccurred())
+	g.Expect(cm.embeddedCertManagerManifestVersion).ToNot(Equal(nil))
+	g.Expect(cm.embeddedCertManagerManifestHash).ToNot(Equal(nil))
```

Comment:
Since these values are strings it makes more sense to check if they are empty instead of nil because they'll technically always be not nil since the default will be `""`. 
```suggestion
	g.Expect(cm.embeddedCertManagerManifestVersion).ToNot(BeEmpty())
	g.Expect(cm.embeddedCertManagerManifestHash).ToNot(BeEmpty())
```

Reply:
thanks for the reminder ~ :) 

will update